packages/subgraph/src/entities/Global.ts

```diff
@@ -135,3 +137,21 @@ export function decreaseNumberOfTrovesClosedByOwner(): void {
   global.numberOfOpenTroves++;
   global.save();
 }
+
+export function handleLQTYStakeChange(stakeChange: LqtyStakeChange): void {
+  let global = getGlobal();
+
+  if (stakeChange.operation == "stakeCreated") {
+    global.totalNumberOfLQTYStakes++;
+    global.totalLQTYTokensStaked = global.totalLQTYTokensStaked.plus(stakeChange.amountChange);
+  } else if (stakeChange.operation == "stakeIncreased") {
+    global.totalLQTYTokensStaked = global.totalLQTYTokensStaked.plus(stakeChange.amountChange);
+  } else if (stakeChange.operation == "stakeDecreased") {
+    global.totalLQTYTokensStaked = global.totalLQTYTokensStaked.minus(stakeChange.amountChange);
+  } else if (stakeChange.operation == "stakeRemoved") {
+    global.totalNumberOfLQTYStakes--;
```

Comment:
I would add a second counter for non-zero stakes instead. For example `totalNumberOfTroves` tracks the number of existing `Trove` entities (open or closed), and `numberOfOpenTroves` is the number of open Troves.

The reason for having counters like this is to be able to build nicer paginators in frontends, since The Graph – even though it supports skipping and limiting – can't return the total number of results in a query (and the most it will return at a time is 1000).

Reply:
Nice I can definitely foresee us wanting the statistic of total number of UNI LP stakes and derivatives of that